### PR TITLE
fix: unable to import PyEVMBackend from fresh upgrade

### PR DESCRIPTION
**Pull request by unparalleled-js**
### What was wrong?

fixes: #212 

### How was it fixed?

Check version when checking if py-evm installed

### To-Do:

- [x] Add entry to the [CHANGELOG](https://github.com/ethereum/eth-tester/blob/master/CHANGELOG)

#### Cute Animal Picture

baby_squirrel

### CODE REVIEW

**Review by kclowes — COMMENTED** [review]
Thanks for the change, you're so fast! This is sort of a nit, but the error messaging where `is_pyevm_available` gets called assumes that pyevm isn't installed (like [here](https://github.com/ethereum/eth-tester/blob/42462659fbb5b2f13c9701ef30ee2712af6632e1/eth_tester/backends/pyevm/main.py#L302), and [here](https://github.com/ethereum/eth-tester/blob/42462659fbb5b2f13c9701ef30ee2712af6632e1/eth_tester/backends/__init__.py#L30)), so I wonder if it would be better to make these two separate checks? Like leave `is_pyevm_available`, and then add a method that checks that the version of py-evm is a high enough version so we can make the error messages slightly better. Or we can update the wording of the error messages. What do you think?

**Comment by unparalleled-js** [comment]
> here

I think what I would do then is change to the error messages to indicate `pyevm >= 5.0.0` is not installed and maybe change the method name to `is_supported_pyevm_version_available()`. I think that would be easier because we will always have to check both anyway.

**Comment by kclowes** [comment]
@unparalleled-js - This is ready to go, right?

**Review by kclowes — APPROVED** [review]

**Comment by unparalleled-js** [comment]
Yeah, unless you know a way to dynamically get the min supported version. I tried to see if we could do that but couldn't find a way.

**Comment by kclowes** [comment]
For [this line](https://github.com/ethereum/eth-tester/pull/213/files#diff-174d7e8e64979634cd2867dce9771d4afe3646617ffde84cd4c90336e4dea669R22)/error messages? Is it set somewhere else?

**Comment by kclowes** [comment]
I felt bad for breaking your CHANGELOG again and again, so I added towncrier to eth-tester. So now there will be newsfragments like our other repos, and less merge conflicts :tada:. You can pull in master to get that update. I'm happy to add the snippet from the CHANGELOG in this PR to a newsfragment if you want, just let me know!

File: eth_tester/backends/pyevm/utils.py [hunk 1]
```diff
@@ -17,10 +17,6 @@ def get_pyevm_version():
...
-def is_pyevm_available():
-    pyevm_version = get_pyevm_version()
-
-    if pyevm_version is None:
-        return False
-    else:
-        return True
+def is_supported_pyevm_version_available():
+    version = get_pyevm_version()
+    return version and version >= Version("0.5.0")
```
Comment (unparalleled-js) [inline eth_tester/backends/pyevm/utils.py:22]: This is the version that would be nice to dynamically get.
I can think of a way to do it but that involves creating a source-of-truth that both here and `setup.py` can use... However, it seems like there should be a way using `pkg_resources` or something but I could not figure out a way.

**Comment by unparalleled-js** [comment]
> I felt bad for breaking your CHANGELOG again and again, so I added towncrier to eth-tester. So now there will be newsfragments like our other repos, and less merge conflicts 🎉. You can pull in master to get that update. I'm happy to add the snippet from the CHANGELOG in this PR to a newsfragment if you want, just let me know!

I think I upgraded to the newsfragments approach correctly!

File: eth_tester/backends/pyevm/utils.py (hunk 1 above)
Comment (kclowes) [reply eth_tester/backends/pyevm/utils.py:22]: Yeah, I don't see a nice way to do it either. I think this is good enough for now 👍

**Pull request merged**